src/sbvr-api/uri-parser.ts

```diff
@@ -263,15 +267,27 @@ export const metadataEndpoints = ['$metadata', '$serviceroot'];
 
 export async function parseOData(
 	b: UnparsedRequest & { _isChangeSet?: false },
+	headers?: {
+		[key: string]: string | string[] | undefined;
+	},
 ): Promise<ParsedODataRequest>;
 export async function parseOData(
 	b: UnparsedRequest & { _isChangeSet: true },
+	headers?: {
+		[key: string]: string | string[] | undefined;
+	},
 ): Promise<ParsedODataRequest[]>;
 export async function parseOData(
 	b: UnparsedRequest,
+	headers?: {
+		[key: string]: string | string[] | undefined;
+	},
 ): Promise<ParsedODataRequest | ParsedODataRequest[]>;
 export async function parseOData(
 	b: UnparsedRequest,
+	headers?: {
```

Comment:
It might be worth naming this as `defaultHeaders` to be a bit clearer what it does